vng_api_common/inspectors/query.py

```diff
@@ -33,8 +33,8 @@ def get_filter_parameters(self, filter_backend):
                 parameter._filter_field = filter_field
 
                 help_text = filter_field.extra.get(
-                    "help_text", model_field.help_text if model_field else ""
-                )
+                    "help_text", None
+                ) or model_field.help_text if model_field else ""
```

Comment:
This only fixes it because `help_text` is specified on the filter field and the `or` expression is not evaluated in the patch. This is probably more robust:

```suggestion
                    "help_text",
                    getattr(model_field, "help_text", "") if model_field else ""
                )
```